### ISSUE TITLE
🐛 Bug Report: Cannot find module '@novu/framework/next' or its corresponding type declarations.ts

### ISSUE THREAD

**Issue by patbeqo** [notifirehq/notifire#6332]
### 📜 Description

Cannot find module '@novu/framework/next' or its corresponding type declarations.ts

### 👟 Reproduction steps
import { serve } from "@novu/framework/next";

"@novu/framework": "2.0.0-canary.5"

### 👍 Expected behavior

Typescript support

### 👎 Actual Behavior with Screenshots

Unable to import from '@novu/framework/next'

### Novu version

2.0.0-canary.5

### npm version

10.5.0

### node version

v18.20.1

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by brunocamboim** [comment]
Did you find any solutions? I have the same problem, but with @novu/framework/express

**Comment by patbeqo** [comment]
@brunocamboim  I've disabled the type checking for now...
```
// @ts-ignore
import { serve } from "@novu/framework/next";
```

**Comment by v1s10n-4** [comment]
I've had to use the complete path to the framework file:
```js
import { serve } from "@novu/framework/dist/servers/next";
```

**Comment by paulwer** [comment]
dont know, if this is helpfull, but in our case this was related to using commonjs modules, so we used require as an workaround and it works with our setup of express.

Maybe a comminjs build of the lib would be suiteable.

**Comment by SokratisVidros** [comment]
@patbeqo which Next.js version are you using? Can you please send us a small reproduction?

**Comment by rifont** [comment]
Hey @patbeqo, we have just released `@novu/framework`@[2.4.0](https://www.npmjs.com/package/@novu/framework/v/2.4.0) with both ESM+CJS bundled into build artifacts 🎉
